### ISSUE TITLE
DSpace 7 OAI persistent delete handling

### ISSUE THREAD

**Issue by kanasznagyzoltan** [DSpace/DSpace#3077]
The DSpace 7 OAI at the moment is not handling the record deletation status. If I delete a record permanently from a collection these deletation is not displayed in the corresponding OAI Set. 
This information should be displayed in the header information of the record.

**Comment by tdonohue** [comment]
This is known "current behavior" (as it also exists in DSpace 6).  Currently, the only way for deletions to be displayed via OAI-PMH is to first withdraw the item (which should give an OAI-PMH deletion record), then permanently delete it.  See also the docs at https://wiki.lyrasis.org/pages/viewpage.action?pageId=68064778

**Comment by tdonohue** [comment]
Small clarification to this ticket.  DSpace's current behavior is to only support `transient` deletions, per the OAI spec: https://www.openarchives.org/OAI/openarchivesprotocol.html#DeletedRecords

So, this ticket is proposing a change to `persistent` deletions. That'd require a new feature, as that status would imply some sort of deletion "tombstone" would need to exist in the OAI-PMH response for deleted records.

**Comment by tdonohue** [comment]
At this time, this feature is waiting on a volunteer to investigate & start to implement.  If a volunteer can be found, then this would move forward immediately.  But, otherwise it will need to wait until either a volunteer is located or its priority changes.

The core dev team is concentrating first on completing the "Tier list" of missing 7.x features (with a concentration on features that existed in 6.x but are not yet in 7.x): https://wiki.lyrasis.org/display/DSPACE/DSpace+Release+7.0+Status#DSpaceRelease7.0Status-Tier1:HighestPriorityTasksfor7.x   Other new features _not_ on that Tier list are therefore lower priority at this time.
